cmd/uhc/cluster/status/status.go

```diff
@@ -0,0 +1,133 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package status
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/openshift-online/uhc-cli/pkg/config"
+	"github.com/openshift-online/uhc-cli/pkg/util"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	debug bool
+}
+
+var Cmd = &cobra.Command{
+	Use:   "status CLUSTERID",
+	Short: "status of a cluster",
+	Long:  "Get the status of a cluster identified by its cluster ID",
+	Run:   run,
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.BoolVar(
+		&args.debug,
+		"debug",
+		false,
+		"Enable debug mode.",
+	)
+}
+func run(cmd *cobra.Command, argv []string) {
+
+	if len(argv) != 1 {
+		fmt.Fprintf(os.Stderr, "Expected exactly one cluster\n")
+		os.Exit(1)
+	}
+
+	clusterID := "id='" + argv[0] + "'"
+
+	// Load the configuration file:
+	cfg, err := config.Load()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't load config file: %v\n", err)
+		os.Exit(1)
+	}
+	if cfg == nil {
+		fmt.Fprintf(os.Stderr, "Not logged in, run the 'login' command\n")
+		os.Exit(1)
+	}
+
+	// Check that the configuration has credentials or tokens that haven't have expired:
+	armed, err := config.Armed(cfg)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't check if tokens have expired: %v\n", err)
+		os.Exit(1)
+	}
+	if !armed {
+		fmt.Fprintf(os.Stderr, "Tokens have expired, run the 'login' command\n")
+		os.Exit(1)
+	}
+
+	// Create the connection:
+	logger, err := util.NewLogger(args.debug)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't create logger: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Create the connection, and remember to close it:
+	connection, err := client.NewConnectionBuilder().
+		Logger(logger).
+		TokenURL(cfg.TokenURL).
+		Client(cfg.ClientID, cfg.ClientSecret).
+		Scopes(cfg.Scopes...).
+		URL(cfg.URL).
+		User(cfg.User, cfg.Password).
+		Tokens(cfg.AccessToken, cfg.RefreshToken).
+		Insecure(cfg.Insecure).
+		Build()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't create connection: %v\n", err)
+		os.Exit(1)
+	}
+	defer connection.Close()
+
+	// Get the client for the resource that manages the collection of clusters:
+	collection := connection.ClustersMgmt().V1().Clusters()
+
+	// Retrieve the collection of clusters:
+	response, err := collection.List().
+		Search(clusterID).
+		Send()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't retrieve clusters: %s", err)
+		os.Exit(1)
+	}
+
+	// Print the result:
+	response.Items().Each(func(cluster *v1.Cluster) bool {
+		clusterMemory := cluster.Memory()
+		clusterCPU := cluster.CPU()
+		clusterStorage := cluster.Storage()
+		fmt.Printf("State:   %s\n"+
+			"Memory:  %.2f/%.2f used\n"+
+			"CPU:     %.2f/%.2f used\n"+
+			"Storage: %.2f/%.2f used\n",
```

Comment:
We are no longer reporting storage for clusters, as the metrics isn't reliable. I will remove it from the SDK. Please remove it from here as well.

Reply:
@jhernand Ok, I will remove this. Do you know of another place this data is exposed? We use this metric in our Dedicated portal today and is something we doing monitoring and alerting on, so a `quick-look` is a nice to have for us.

Reply:
The problem is that the metrics that the clusters report via telemetry aren't specific for the cloud volumes, which is what the users will most probably want to see. Instead we get the amount of storage in the mount points, which isn't reliable. That is why we are removing it. There is no other place where this is available in UHC. It could be possible to get them from the cluster directly. You could connect to the cluster, using the _kubeconfig_ available in `/api/clusters_mgmt/v1/clusters/123/credentials`, but it is more complicated. I'd suggest that if you want to do that you do it in a different pull request.

Reply:
Ack, I agree we can take it to another PR or even a different tool.